spotinst/resource_spotinst_ocean_spark.go

```diff
@@ -24,6 +25,8 @@ import (
 
 const (
 	ErrCodeResourceDoesNotExist = "RESOURCE_DOES_NOT_EXIST"
+	deleteTimeout               = 10 * time.Minute
+	sleepBetweenDeleteChecks    = 1 * time.Minute
```

Comment:
```suggestion
	sleepBetweenDeleteChecks    = 30 * time.Second
```

can be a little bit more aggressive?